### DISCUSSION TITLE
Ignored pull requests?

### DISCUSSION THREAD

**Discussion by yankl** [getgrav/grav#3455] [Development]
Hi, 

Earlier this year I made two pull requests -- one in the main Grav repo (#3336) and [one ](https://github.com/getgrav/grav-plugin-langswitcher/pull/60) in the langswitcher plugin. They're both very minor fixes, just a line or two of code, so they don't seem like they would require much resources to review. 

I'm very much an amateur developer, and have very little experience on Github. (I only came to make these pull requests since I need them to fix issues on my site, without workaround hacks.) So my question is -- is there something else I could do to draw attention to them, or just keep waiting? For instance, I could tag someone but don't want to break any Github etiquette rules. Or are they more likely to be ignored if they're not connected to an issue in the issue tracker? By the looks of it, most of the open pull requests have at least had some discussion on them happen, so I'm wondering if I'm doing something wrong, though I do see on the langswitcher plugin repo sometimes years can go by with no response to a pull request. 

It would be great, if they are indeed worthy, to have them merged, since the particular Grav website I'm developing would be much improved with these fixes. 

I'm also asking with regards to how to proceed with possible future pull requests, as it is somewhat discouraging that the only two I've made so far have been ignored.

Thanks for any help!

**Comment by hughbris** [comment]
I also have a few PRs and issues stalled or ignored for different reasons, many of them valid. In both core and plugins.

The word that needs to be emphasised in your question is "discouraging". I often feel that too. I do appreciate that the developers do an amazing job and have produced an excellent product largely for very little reward of any kind. However, I would have thought that responding positively to help is one way out of the endless cycle of graft and thanklessness.

As a side note, many potentially useful PRs become unmergeable when they become stale. A few times, I've seen other PRs accepted afterwards that do exactly what mine did. That's quite dispiriting and seems arbitrary.

Anyway, I don't really want to rant. I love this project and CMS, fully respect its developers, but I noticed this thread and wanted to add my voice as I think this kind of problem is holding it back.

Let's suggest some positive steps. Mine is an issue triage by the community. Even labeling/prioritising. I suspect many issues can't be resolved by the community though as they need decisions from the core team. The exercise should help and focus our minds though.

Sorry in advance if I've been too blunt.

**Comment by mahagr** [comment]
I think the best way is to just make a comment to the pull request/issue as it may just have been forgotten (I'm too busy now, I will check it later). 

Sometimes it also helps if you provide an easy way to test the issue and some thoughts of what the issue is and the idea behind the fix.

Many times even small fixes may have some unexpected side effects, so they may not be as small as you thought. Or the approach isn't quite right.

In short: It's likely that the issues have just been ignored as nobody had time to check it when the notification came and it was forgotten afterward. Friendly reminder never hurts!
